src/Components/SystemDetail/RecommendationsTable.js

```diff
@@ -9,147 +17,149 @@ import { TextContent, Text, TextVariants } from '@patternfly/react-core';
-    const { resolution, reason, detected_issues: detectedIssues, current_instance: currentInstance, suggested_instances: suggestedInstances,
-        psi_enabled: psiEnabled } = row;
+class RecommendationsTable extends React.Component {
+    constructor(props) {
+        super(props);
+        this.state = {
+            expanded: true
+        };
+    }
+
+    onExpandToggle = () => {
+        this.setState({ expanded: !this.state.expanded });
+    };
+
+    renderExpandedView = (row, index) => {
+        const { description, resolution, reason, detected_issues: detectedIssues,
+            current_instance: currentInstance, suggested_instances: suggestedInstances,
+            psi_enabled: psiEnabled } = row;
 
-    return (
-        <TextContent>
-            <Text component={TextVariants.p}>
-                <Text className="margin-text-bottom"><BullseyeIcon/><strong className="strong-tag-style">Detected issues</strong></Text>
-                {reason}
-            </Text>
-            { detectedIssues && <TableComposable
-                arial-label="Detected issues table"
-                variant="compact"
-                borders={false}
-                className="table-border-top detected-issues-table"
-            >
-                <Thead>
+        return (
+            <Table aria-label="Nested column headers with expandable rows table" variant="compact" className="ros-recommendations-table" >
+                <Thead >
                     <Tr>
-                        <Th>Identified issues by ROS</Th>
+                        <Th />
+                        <Th>
+                      Name
+                        </Th>
                     </Tr>
                 </Thead>
-                <Tbody>
-                    <Tr>
-                        <Td dataLabel="Identified issues by ROS" className="newline tab">
-                            {detectedIssues}</Td>
-                    </Tr>
-                </Tbody>
-            </TableComposable> }
-            <hr/>
-            <Text component={TextVariants.p}>
-                <Text className="margin-text-bottom"><WrenchIcon/><strong className="strong-tag-style">Suggestion</strong></Text>
-                {resolution}
-            </Text>
-            { currentInstance && suggestedInstances && <TableComposable
-                arial-label="Suggestions table"
-                variant="compact"
-                borders={false}
-                className="table-border-top suggestions-table"
-            >
-                <Thead>
+                <Tbody key={'Developer program 1'} isExpanded={this.state.expanded}>
                     <Tr>
-                        <Th>Instance</Th>
-                        <Th>Suggested instances that fit the load better</Th>
+                        <Td expand={{
+                            rowIndex: index,
+                            isExpanded: this.state.expanded,
+                            onToggle: () => this.onExpandToggle(),
```

Comment:
This line can be converted to `onToggle: this.onExpandToggle`
The onToggle property should get a function which will be evoked when the toggle event is triggered, therefore there is no need to create an arrow function that calls this.onExpandToggle function in this case
If I am not mistaken 😇

Reply:
@LiorKGOW Thanks for picking this, I have updated the code in the latest commit. 🙌🏻